lib/backend/compat/compat.go

```diff
@@ -144,6 +150,11 @@ func (c *ModelAdaptor) Apply(d *model.KVPair) (*model.KVPair, error) {
 			return d, nil
 		}
 	case model.NodeKey:
+		// Quick fix for k8s Node call.  The k8s Node.Apply call requires a complete model.Node object,
+		// therefore we do not need to break down the node object.
```

Comment:
Just a thought - what's the behavior if the k8s call fails (e.g. transient connection issue)?  At that point we'll drop into trying to apply the separate components....  which I assume k8s will fail because it won't recognize the individual component types?